crates/viewer/re_view_dataframe/src/view_query/ui.rs

```diff
@@ -283,9 +297,13 @@ impl Query {
                     egui::ComboBox::new("pov_component", "")
                         .selected_text(filter_component.short_name())
                         .show_ui(ui, |ui| {
-                            for component in all_components {
-                                let label = component.short_name();
-                                ui.selectable_value(&mut filter_component, component, label);
+                            for descr in all_components {
```

Comment:
I think we should mark up this and above `.component_name` sites with our usual todo comment, they're hard to spot even when removing `ComponentNameSet`